pages/_app.tsx

```diff
@@ -10,7 +10,6 @@ export default function App({ Component, pageProps }: AppProps) {
   return (
     <>
       <GlobalStyle />
-      <StyledTestHEHE>im red</StyledTestHEHE>
```

Comment:
can we also please remove where `StyledTestHEHE` is defined on line 6?

Reply:
Oops sorry, good point!!